docker/Dockerfile.geth

```diff
@@ -1,4 +1,4 @@
-FROM ethereum/client-go:v1.10.15
+FROM ethereum/client-go
```

Comment:
We decided to hardcode a version for `ethereum/client-go` in order to have always the same deployment and to not be dependent on changes in `ethereum/client-go` versions. This has been a source of issues in the past.
Therefore, I would setup a concrete version (to decode with `zkNode` team).
If needed, the version will be updated.